lib/nfc/protocols/st25tb/st25tb_poller_i.c

```diff
@@ -0,0 +1,297 @@
+#include "st25tb_poller_i.h"
+
+#include "bit_buffer.h"
+#include "core/core_defines.h"
+#include "protocols/st25tb/st25tb.h"
+#include <nfc/helpers/iso14443_crc.h>
+
+#define TAG "ST25TBPoller"
+
+static St25tbError st25tb_poller_process_error(NfcError error) {
+    switch(error) {
+    case NfcErrorNone:
+        return St25tbErrorNone;
+    case NfcErrorTimeout:
+        return St25tbErrorTimeout;
+    default:
+        return St25tbErrorNotPresent;
+    }
+}
+
+static St25tbError st25tb_poller_prepare_trx(St25tbPoller* instance) {
+    furi_assert(instance);
+
+    if(instance->state == St25tbPollerStateIdle) {
+        return st25tb_poller_async_activate(instance, NULL);
+    }
+
+    return St25tbErrorNone;
+}
+
+static St25tbError st25tb_poller_frame_exchange(
+    St25tbPoller* instance,
+    const BitBuffer* tx_buffer,
+    BitBuffer* rx_buffer,
+    uint32_t fwt) {
+    furi_assert(instance);
+
+    const size_t tx_bytes = bit_buffer_get_size_bytes(tx_buffer);
+    furi_assert(
+        tx_bytes <= bit_buffer_get_capacity_bytes(instance->tx_buffer) - ISO14443_CRC_SIZE);
+
+    bit_buffer_copy(instance->tx_buffer, tx_buffer);
+    iso14443_crc_append(Iso14443CrcTypeB, instance->tx_buffer);
+
+    St25tbError ret = St25tbErrorNone;
+
+    do {
+        NfcError error =
+            nfc_poller_trx(instance->nfc, instance->tx_buffer, instance->rx_buffer, fwt);
+        if(error != NfcErrorNone) {
+            FURI_LOG_E(TAG, "error during trx: %d", error);
```

Comment:
Please, change to FURI_LOG_D()
If card is not present, there will be timeout error, which is normal situation and shouldn't be logged in Error log level

Reply:
Fixed in 12533fb2aff0fbf54d692e2b61efd777cb00516a